src/searcher_frame_document.rs

```diff
@@ -95,37 +83,37 @@ impl StatSearcher {
     /// Returns `SearchResult` object.
     ///
     /// Raises a ValueError if there was an error with the search.
-    #[pyo3(signature = (query, allowed_frame_ids))]
+    #[pyo3(signature = (query, filter_fastfield_name=None, filter_fastfield_values=None))]
```

Comment:
# THIS CHANGE WILL REQUIRE CHANGES IN CHRYSALIS CODE

Very similar to changes in the tests here.  Basically, it is no longer assumed that the fastfield for filtering will be called `frame_id__`: you have to supply the name of the fastfield you want the set of ints to be applied against.

In exchange for the extra effort of changing chrys code, we get the ability to search with a filter on any of the fastfields, i.e. doc or frame or sentence, etc.  This could be quite handy.